fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraLdpIT.java

```diff
@@ -938,7 +937,7 @@ public void testGetRepositoryGraph() throws IOException {
         try (final CloseableGraphStore graph = getGraphStore(getObjMethod(""))) {
             logger.trace("Retrieved repository graph:\n" + graph);
             assertTrue("Should find the root type", graph.contains(ANY,
-                    ANY, HAS_PRIMARY_TYPE.asNode(), createLiteral(ROOT)));
+                    ANY, type.asNode(), createURI(MODE_NAMESPACE + "root")));
```

Comment:
:+1: -- having a fedora:Root type seems like a good way to explicitly mark the root node.  I would also be fine with just omitting it (lacking a parent is also a good implicit marker of root-ness).